core/src/main/java/com/microsoft/applicationinsights/internal/heartbeat/DefaultHeartbeatPropertyProvider.java

```diff
@@ -145,7 +152,7 @@ private String getSdkVersion() {
    * Gets the OS version on which application is running.
    * @return String representing OS version
    */
-  private String getOsType() {
+  private String getOsVersion() {
```

Comment:
Do we want the OS version? If so, you should explore the difference between `os.name` and `os.version` (i.e. `os.name` does not contain version information).

I also found a few articles stating that `os.name` doesn't have the correct value when on Windows 10. Perhaps there is a different way to find this information.

Reply:
We can amend it, if there is a better way though I am not aware of one. This is meta data which is not very very vital though.